src/v2/providers/pubsub.ts

```diff
@@ -86,29 +89,30 @@ export class Message<T> {
   }
 }
 
-/** The interface published in a Pub/Sub publish subscription. */
+/** @beta The interface published in a Pub/Sub publish subscription. */
 export interface MessagePublishedData<T = any> {
   readonly message: Message<T>;
   readonly subscription: string;
 }
 
-/** PubSubOptions extend EventHandlerOptions but must include a topic. */
+/** @beta PubSubOptions extend EventHandlerOptions but must include a topic. */
 export interface PubSubOptions extends options.EventHandlerOptions {
   topic: string;
 }
 
-/** Handle a message being published to a Pub/Sub topic. */
+/** @beta Handle a message being published to a Pub/Sub topic. */
```

Comment:
Here and the other overload the params and type param need to be documented

Reply:
Done!